lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/view/FindMealActivity.kt

```diff
@@ -17,52 +27,148 @@ import kotlinx.coroutines.launch
 
 class FindMealActivity : AppCompatActivity() {
     private val mainScope = MainScope()
+    private val sdk = MealsSDK()
+    private val mealsAdapter = MealsAdapter(listOf())
 
     private lateinit var mealsRecyclerView: RecyclerView
     private lateinit var progressBarView: FrameLayout
     private lateinit var swipeRefreshLayout: SwipeRefreshLayout
 
-    private val sdk = MealsSDK()
-    private val mealsAdapter = MealsAdapter(listOf())
+    private lateinit var fusedLocationProviderClient: FusedLocationProviderClient
```

Comment:
If you want to avoid lateinit and make it immutable you can use `private val fusedLocationProviderClient: FusedLocationProviderClient by lazy { LocationServices.getFusedLocationProviderClient(this) }`

That's the same as doing `lazy var` in Swift and means it will only call getFusedLocationProviderClient when first accessed, and has the added bonus of making it immutable and not lateinit

Reply:
Nice, I like lazy :-)